website/source/docs/job-specification/affinity.html.md

```diff
@@ -28,9 +28,39 @@ be expressed on [attributes][interpolation] or [client metadata][client-meta].
 Additionally affinities may be specified at the [job][job], [group][group], or
 [task][task] levels for ultimate flexibility.
 
-Placing the same affinity at both the job level and at the group level is redundant
-since affinities are applied hierarchically. The job affinities will affect
-all groups (and tasks) in the job.
+```hcl
+job "docs" {
+  # Prefer nodes in the us-west1 datacenter
```

Comment:
I think this example is slightly confusing.

`us-west1` would in most setups be a region, with `us-west1[a|b|c|...]` being the DC within that region